File: UnicoVehicle/UnicoVehicle/UnicoVehicle.DTO/Miscellaneous/Showroom.cs
using System;
namespace UnicoVehicle.DTO.Miscellaneous
{
    public class Showroom
    {
        public int ShowroomId { get; set; }
        public Company Company { get; set; }
        public District District { get; set; }
        public string ShowroomName { get; set; }
    }
}

File: Flutter App/unicoVehicle/lib/Models/Miscellaneous/Showroom.dart
import './Company.dart';
import './District.dart';

class Showroom {
  final int showroomId;
  final Company company;
  final District district;
  final String showroomName;

  Showroom({
    this.company,
    this.district,
    this.showroomId,
    this.showroomName,
  });
}
